sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineScaleSetImpl.java

```diff
@@ -609,6 +609,15 @@ public VirtualMachineScaleSetNetworkInterface getNetworkInterfaceByInstanceId(St
             .getByVirtualMachineScaleSetInstanceId(this.resourceGroupName(), this.name(), instanceId, name);
     }
 
+    @Override
```

Comment:
Have a test to cover this interface?

Reply:
Tested locally, but not added to test, as basically parallel code.

Reply:
Added the one line test on the async.